src/Main.elm

```diff
@@ -64,7 +57,7 @@ main =
         { init = \_ -> init
         , view = view
         , update = update
-        , subscriptions = subscriptions
+        , subscriptions = \_ -> Sub.none
```

Comment:
My thinking for a starter repo was that it might be easier for people to have it fully written out so they can see the type signatures and change it more easily. Generally I would write it this way in my own code as well, but I think a starter repo is a bit of a special case since it's trying to lay out a template that people can easily hook into.

What do you think about moving this back out to a named function?

Reply:
Discussed on slack, this is a tricky balance between useful template code and good practice. Happy to restore it understanding that it's a template.